src/operator/main.go

```diff
@@ -230,20 +217,24 @@ func main() {
 		logrus.WithError(err).Fatal("unable to set up ready check")
 	}
 
-	ctx, cancelFunc := context.WithTimeout(context.Background(), 10*time.Second)
+	signalHandlerCtx := ctrl.SetupSignalHandler()
+	timeoutCtx, cancelFunc := context.WithTimeout(signalHandlerCtx, 10*time.Second)
 	defer cancelFunc()
-	cloudClient, connectedToCloud, err := otterizecloud.NewClient(ctx)
+	otterizeCloudClient, connectedToCloud, err := otterizecloud.NewClient(timeoutCtx)
 	if err != nil {
-		logrus.WithError(err).Warning("Failed to connect to Otterize cloud")
+		logrus.WithError(err).Fatal("Failed to initialize Otterize Cloud client")
```

Comment:
IMO the operator shouldn't crash if it cannot connect to the cloud. Operator downtime means potential access issues between the services in the cluster, so it should be as resistant as possible to cloud connectivity loss.